spec.html

```diff
@@ -1030,7 +1030,13 @@ <h1>Mathematical Operations</h1>
+    </emu-clause>
+
+    <emu-clause id="sec-identity">
+      <h1>Identity</h1>
+      <p>In this specification, the word &ldquo;is&rdquo; is used to identify two values, as in &ldquo;If _bool_ is *true*, then&rdquo;. This identification is different when applied to values which have identity than it is when applied to those without identity.</p>
+      <p><dfn variants="values without identity,value without identity">Values without identity</dfn> are identified by the set of all of their innate qualities &mdash; qualities such as the magnitude of an integer or the length of a sequence. Because of this, values without identity may be manifest anywhere simply by fully describing their qualities. It is not meaningful to modify a value that does not have identity. Examples of values without identity include, but are not limited to: <emu-xref href="#sec-ecmascript-language-types-boolean-type">Booleans</emu-xref>; mathematical values and extended mathematical values; <emu-xref href="#sec-ecmascript-language-types-number-type">Numbers</emu-xref>; <emu-xref href="#sec-ecmascript-language-types-bigint-type">BigInts</emu-xref>; *null*; *undefined*; sequences, including <emu-xref href="#sec-ecmascript-language-types-string-type">Strings</emu-xref>, ECMAScript source text, surrogate pairs, Directive Prologues, etc; UTF-16 code units; Unicode code points; enums; abstract operations, including syntax-directed operations, host hooks, etc; and ordered pairs.</p>
+      <p>In contrast, <dfn variants="value with identity">values with identity</dfn> are identified not only by their innate qualities but also another unobservable quality that makes each such value unique. Values with identity may be thought of as a value without identity paired with an unguessable, unchangeable, universally-unique number: the <em>identity</em>. References to existing values with identity cannot be manifest simply by describing them, as the identity itself is indescribable; instead, references to these values must be explicitly passed from one place to another. Values with identity can have their qualities modified in-place, causing all holders of the value to immediately observe the new qualities. Examples of values with identity include, but are not limited to: <emu-xref href="#sec-object-type">Objects</emu-xref>, including function objects, exotic objects, etc; any kind of Records, including Property Descriptors, PrivateElements, etc; <emu-xref href="#sec-ecmascript-language-types-symbol-type">symbols</emu-xref>; Parse Nodes; Lists; <emu-xref href="#sec-set-and-relation-specification-type">Sets</emu-xref> and Relations; Abstract Closures; Data Blocks; Private Names; execution contexts and execution context stacks; agent signifiers; and WaiterLists.</p>
```

Comment:
I'm slightly confused by the inclusion of symbols, parse nodes, private names and agent signifiers. All of these are immutable, they cannot "*have their qualities modified in-place*". Yes, they do have an "*unobservable quality that makes each such value unique*", but is that an "identity"? Specifically, they can be serialised and copied and do not need to have their references shared, as long as the "*unguessable, unchangeable, universally-unique number*" is copied with them.

I would prefer extra elaboration, possibly even a trichotomy instead of a dichotomy.

Reply:
Maybe just add "mutable":
```suggestion
      <p>In contrast, <dfn variants="value with identity">values with identity</dfn> are identified not only by their innate qualities but also another unobservable quality that makes each such value unique. Values with identity may be thought of as a value without identity paired with an unguessable, unchangeable, universally-unique number: the <em>identity</em>. References to mutable values with identity cannot be manifest simply by describing them, as the identity itself is indescribable; instead, references to these values must be explicitly passed from one place to another. Mutable values with identity can have their qualities modified in-place, causing all holders of the value to immediately observe the new qualities. Examples of values with identity include, but are not limited to: <emu-xref href="#sec-object-type">Objects</emu-xref>, including function objects, exotic objects, etc; any kind of Records, including Property Descriptors, PrivateElements, etc; <emu-xref href="#sec-ecmascript-language-types-symbol-type">symbols</emu-xref>; Parse Nodes; Lists; <emu-xref href="#sec-set-and-relation-specification-type">Sets</emu-xref> and Relations; Abstract Closures; Data Blocks; Private Names; execution contexts and execution context stacks; agent signifiers; and WaiterLists.</p>
```

Reply:
I agree with this comment. I think I'd say "Some values with identity are mutable, that is, they can have their qualities modified in place..."